src/App.js

```diff
@@ -39,32 +68,65 @@ function App() {
     []
   );
   return (
-    <div className="App">
-      <button disabled={state.show === "all"} onClick={changeView("all")}>
+    <div
+      className="App"
+      css={{
+        background: 'linear-gradient(#e6646533, #9198e588)',
+        height: '100%'
+      }}
+    >
+      <Select disabled={state.show === 'all'} onClick={changeView('all')}>
         All
-      </button>
-      <button disabled={state.show === "done"} onClick={changeView("done")}>
+      </Select>
+      <Select disabled={state.show === 'done'} onClick={changeView('done')}>
         Done
-      </button>
-      <button disabled={state.show === "undone"} onClick={changeView("undone")}>
+      </Select>
+      <Select disabled={state.show === 'undone'} onClick={changeView('undone')}>
         Undone
+      </Select>
+      <br/>
+      <div>
+        {state.tasks
+          .filter(x =>
+            state.show === 'all' ? true : state.show === 'done' ? x.done : !x.done
+          )
+          .map(x => (
+            <TodoItem
+              id={x.id}
+              name={x.name}
+              remove={remove(x.id)}
+              check={check(x.id)}
```

Comment:
Even handling props like these are better named `onRemove` and `onCheck` so that we can distinguish them from normal props.